dcc-submission-loader/src/test/java/org/icgc/dcc/submission/loader/util/AbstractPostgressTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2016 The Ontario Institute for Cancer Research. All rights reserved.                             
+ *                                                                                                               
+ * This program and the accompanying materials are made available under the terms of the GNU Public License v3.0.
+ * You should have received a copy of the GNU General Public License along with                                  
+ * this program. If not, see <http://www.gnu.org/licenses/>.                                                     
+ *                                                                                                               
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY                           
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES                          
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT                           
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,                                
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED                          
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;                               
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER                              
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN                         
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.icgc.dcc.submission.loader.util;
+
+import javax.sql.DataSource;
+
+import lombok.val;
+
+import org.junit.After;
+import org.junit.Before;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import ru.yandex.qatools.embed.postgresql.PostgresProcess;
+import ru.yandex.qatools.embed.postgresql.PostgresStarter;
+import ru.yandex.qatools.embed.postgresql.config.PostgresConfig;
+
+public abstract class AbstractPostgressTest {
+
+  protected DataSource dataSource;
+  protected JdbcTemplate jdbcTemplate;
+  protected PostgresConfig config;
+
+  private PostgresProcess process;
+
+  @Before
+  public void setUp() throws Exception {
+    val runtime = PostgresStarter.getDefaultInstance();
```

Comment:
I think this should be moved to its own class. See 

- https://github.com/icgc-dcc/dcc-submission/blob/d7f6ca0a7c3ab68eac59dd8dd1d6dc370ee75d20/dcc-submission-server/src/test/java/org/icgc/dcc/submission/repository/AbstractRepositoryTest.java
- https://github.com/icgc-dcc/dcc-common/blob/2787c6f3ea2ab4819257ada0d9945abd37255b9c/dcc-common-test/src/main/java/org/icgc/dcc/common/test/mongodb/EmbeddedMongo.java